pmd-core/etc/xslt/pmd-report-v2.xslt

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+xmlns:scan="http://pmd.sourceforge.net/report/2.0.0"
+  >
+
+<xsl:template match="scan:pmd">
+  <xsl:variable name="version" select="@version" />
+  <xsl:variable name="timestamp" select="@timestamp" />
+  <xsl:variable name="count">0</xsl:variable>
+  <xsl:variable name="total">0</xsl:variable>
+
+
+  <html>
+   <head>
+   <title>PMD Report </title>
```

Comment:
Nit: the indentation is not consistent - some lines are indented correctly, some are not...